README.md

```diff
@@ -174,6 +174,20 @@ module "app_ecs_service" {
 
 ## Upgrade Path
 
+### 4.x to 5.0.0
+
+In versions 4.x and prior, the following resources existed as arrays (toggled by a `count` meta-argument). With 5.0.0, each pair has been merged into a single resource.
+
+* `aws_cloudwatch_metric_alarm.alarm_cpu[0]` xor `aws_cloudwatch_metric_alarm.alarm_cpu_no_lb[0]` -> `aws_cloudwatch_metric_alarm.alarm_cpu`
+* `aws_cloudwatch_metric_alarm.alarm_mem[0]` xor `aws_cloudwatch_metric_alarm.alarm_mem_no_lb[0]` -> `aws_cloudwatch_metric_alarm.alarm_cpu`
+* `aws_ecs_service.main[0]` xor `aws_ecs_service.main_no_lb[0]` -> `aws_ecs_service.main`
+
+To upgrade to 5.0.0, you will need to perform a `terraform mv` for any affected resources to avoid destruction and recreation. Alternatively, you can let Terraform delete/recreate the deployed resources.
```

Comment:
```suggestion
To upgrade to 5.0.0, you will need to perform a `terraform state mv` for any affected resources to avoid destruction and recreation. Alternatively, you can let Terraform delete/recreate the deployed resources.
```

You might want to give a full command example, eg:

```bash
$ terraform state mv module.example.aws_cloudwatch_metric_alarm.alarm_cpu[0] module.example.aws_cloudwatch_metric_alarm.alarm_cpu
```